pages/home/script.js

```diff
@@ -244,12 +244,12 @@ const getReportApiCall = async (lat, lng, size = 1000, cursor = 0) => {
   // clear previous reports
   reports = [];
 
+  // search position
   const positionChange = searchInput.dataset.positionChange === 'true';
-  //TEMP
-  // const url = `hazard-report?cursor=${cursor}&size=${size}&lat=${
-  //   positionChange ? positionSecondary.lat : lat
-  // }&lng=${positionChange ? positionSecondary.lng : lng}`;
-  const url = `hazard-report`;
+
+  const url = `hazard-report?cursor=${cursor}&size=${size}&lat=${
```

Comment:
remove the cursor and size from the request, since these params are disabled in the backend we don't need them.